man/man8/zpool.8

```diff
@@ -520,8 +520,9 @@ If a read error is encountered on a cache device, that read I/O is reissued to
 the original storage pool device, which might be part of a mirrored or raidz
 configuration.
 .Pp
-The content of the cache devices is considered volatile, as is the case with
-other system caches.
+The content of the cache devices is can be made non-volatile, see the
```

Comment:
Typo, remove the `is`:
`The content of the cache devices can be made non-volatile, see the`